### DISCUSSION TITLE
ant-design-web3 slogan or description

### DISCUSSION THREAD

**Discussion by jeasonstudio** [ant-design/ant-design-web3#613] [Polls]
![image](https://github.com/ant-design/ant-design-web3/assets/17971291/85a53c40-edc9-4e57-8ee5-699f0044a3ef)

需要定下来一句话描述，可以是介绍、口号等，下面列了几个选项，大家如果有更好的想法欢迎提出 👇

**Comment by yutingzhao1991** [comment]
我比较喜欢 `Awesome react components for building dapps easier. `，但是它的 `Awesome` 换一个词感觉更好，`Awesome` 用来自称感觉太高调了。

比如换成类似 `实用的` 这样的描述感觉会更好。

**Reply by kiner-tang** [reply]
或者是 `高效的`？

**Reply by yutingzhao1991** [reply]
嗯，好像不错

**Comment by yutingzhao1991** [comment]
Useful react components for building dapps easier. 

实用的 React 组件库，助你更轻松地构建你的去中心化应用。

**Comment by kiner-tang** [comment]
Efficient react components for building dapps easier.
高效的 React 组件库，助你更轻松地构建你的去中心化应用。

**Reply by jeasonstudio** [reply]
我也觉得 efficient 更贴切一些

**Reply by yutingzhao1991** [reply]
嗯，这样应该 OK。就是看看英语语法上面对不对，别搞出个中式英文，我找人问问。

**Reply by yutingzhao1991** [reply]
问了下威廉，没问题。

**Comment by yutingzhao1991** [comment]
英文的用法我问了下 GPT，
![image](https://github.com/ant-design/ant-design-web3/assets/1061968/0394930a-3cf5-4ea7-8eb3-42755d6a2858)

`React Components for Simplifying dApp Development` 好像这样写更符合习惯？

**Reply by jeasonstudio** [reply]
可以用逗号把长难句拆开，比如 `Efficient react components library, make dapps easier to build.`

**Reply by kiner-tang** [reply]
赞成这个

**Reply by yutingzhao1991** [reply]
嗯，这个我也问问英文选手，看看哪个更好

**Reply by yutingzhao1991** [reply]
![image](https://github.com/ant-design/ant-design-web3/assets/1061968/1fffa062-e5f0-4646-8442-8cb9feef5e0d)

感觉没有分开的必要。

- An efficient React component library that simplifies the development of dapps.
- Efficient react components for building dapps easier.

看看上面两个哪个好。

**Reply by kiner-tang** [reply]
我投后面那个，又能说明亮点，又简介明了

**Reply by yutingzhao1991** [reply]
嗯，前面这个可能就更口语化了。

**Reply by jeasonstudio** [reply]
那就这么定下来吧，犹豫就会败北

**Reply by yutingzhao1991** [reply]
那就 `Efficient react components for building dapps easier.` ？ @kiner-tang @jeasonstudio 我们三人就拍了

**Comment by yutingzhao1991** [comment]
@afc163 的 `react components born for dapps`  这个看上去也不错，可以留在 2.0 升级用。
